src/objects/zcl_abapgit_object_clas.clas.abap

```diff
@@ -9,27 +9,68 @@ CLASS zcl_abapgit_object_clas DEFINITION PUBLIC INHERITING FROM zcl_abapgit_obje
         iv_language TYPE spras.
 
   PROTECTED SECTION.
-    DATA: mi_object_oriented_object_fct TYPE REF TO zif_abapgit_oo_object_fnc,
-          mv_skip_testclass             TYPE abap_bool.
-    METHODS:
-      deserialize_abap
-        IMPORTING io_xml     TYPE REF TO zcl_abapgit_xml_input
-                  iv_package TYPE devclass
-        RAISING   zcx_abapgit_exception,
-      deserialize_docu
-        IMPORTING io_xml TYPE REF TO zcl_abapgit_xml_input
-        RAISING   zcx_abapgit_exception,
-      deserialize_tpool
-        IMPORTING io_xml TYPE REF TO zcl_abapgit_xml_input
-        RAISING   zcx_abapgit_exception,
-      deserialize_sotr
-        IMPORTING io_xml     TYPE REF TO zcl_abapgit_xml_input
-                  iv_package TYPE devclass
-        RAISING   zcx_abapgit_exception,
-      serialize_xml
-        IMPORTING io_xml TYPE REF TO zcl_abapgit_xml_output
-        RAISING   zcx_abapgit_exception.
 
+    DATA mi_object_oriented_object_fct TYPE REF TO zif_abapgit_oo_object_fnc .
+    DATA mv_skip_testclass TYPE abap_bool .
+    DATA mt_langu_additional TYPE zif_abapgit_lang_definitions=>tt_langu .
+    DATA mv_classpool_name TYPE progname .
+
+    METHODS deserialize_abap
```

Comment:
please move this pretty printing to a separate pull request

Reply:
I restore that class to do only minimum changes needed